docs/variants/qemu_q35/initial-deployment.md

```diff
@@ -1,19 +1,20 @@
 # Initial Deployment
 
-Initial deployment of Dasharo (UEFI) firmware on QEMU Q35 does not need any
-special actions. QEMU Q35 is an emulated platform, not real hardware.
+Initial deployment of Dasharo (coreboot+UEFI) firmware on QEMU Q35 does not need
+any special actions. QEMU Q35 is an emulated platform, not real hardware.
 
-Running the Dasharo (UEFI) is as simple as invoking a QEMU command:
+Running the Dasharo (coreboot+UEFI) is as simple as invoking a QEMU command:
 
 ```bash
 qemu-system-x86_64 -machine q35,smm=on \
-	-global driver=cfi.pflash01,property=secure,value=on \
-	-drive if=pflash,format=raw,unit=0,file=Build/OvmfX64/RELEASE_GCC5/FV/OVMF_CODE.fd,readonly=on \
-	-drive if=pflash,format=raw,unit=1,file=Build/OvmfX64/RELEASE_GCC5/FV/OVMF_VARS.fd \
-	-debugcon file:debug.log -global isa-debugcon.iobase=0x402 \
-	-global ICH9-LPC.disable_s3=1
+	-drive if=pflash,format=raw,file=build/coreboot.rom
 ```
 
+This is the minimal set of parameters that are always required, but you may add
+more to e.g. connect additional devices or redirect serial output to file. Refer
```

Comment:
nit, but really commenting to say that `-debugcon` worked for OVMF, but doesn't work in case of this firmware.
```suggestion
more to e.g. connect additional devices or redirect serial output to a file. Refer
```

Reply:
https://github.com/Dasharo/docs/compare/2a47bdd5f3eae81d35cc9545010489747bac76c6..fd1fc06fa12f4d9dde6a66b01a567657c6bb1a5d

Is it worth to bring support for `-debugcon`? Everything can be printed on serial, and that way it would be easier to synchronize with coreboot's output, e.g. from SMI handlers.

Reply:
I remember coreboot stalling trying to use `-debugcon` while I was testing update capsules, but I didn't notice it on newer versions of the changes.  I think `-debugcon` can be faster than serial, but don't know if it's actually noticeable, so maybe there are no advantages.